site/src/pages/components/radio.research.mdx

```diff
@@ -38,35 +39,35 @@ After the research there are still [discussions to be made](#discussion), where
 
 ## Anatomy
 
-<Anatomy component="Radio Button" />
+<Anatomy component="Radio" />
 
 ## Concepts
 
-<Concepts client:load component="Radio Button" />
+<Concepts client:load component="Radio" />
 
 ## Discussion
 
-At a point during the research, the scope of this study os shifted from the “radio button” to the “radio button group”, because there is no particular strong use case for a standalone “radio button”, and also because this use case is against the W3 Specification, the actual component to be considered is the “radio button group”, being composed of two or more radio buttons.
+At a point during the research, the scope of this study os shifted from the “radio” to the “radio group”, because there is no particular strong use case for a standalone “radio”, and also because this use case is against the W3 Specification, the actual component to be considered is the “radio group”, being composed of two or more radios.
 
 Also based on this fact and on the current state of the design systems, a more final anatomy was proposed.
 
 ### Proposed Anatomy - by Tag Hierarchy
 
-This section shows the proposed visual constituents for the _Radio Button Group_, which is here considered to be the actual component, while the radio buttons are a part of the group.
+This section shows the proposed visual constituents for the _Radio Group_, which is here considered to be the actual component, while the radios are a part of the group.
 
-- Radio Button Group (the actual component)
+- Radio Group (the actual component)
   - Group Label (functions as a label for the whole group)
   - Required Indicator
-  - Radio Button (at least two required)
+  - Radio (at least two required)
     - Checked Indicator (input)
     - Label (optional)
-  - Radio Button
+  - Radio
     - and so on...
   - and so on...
 
 #### The problem
 
-The above anatomy is regular enough to be assembled in a way similar to (but more complex than) a `<select>`. Attributes set on the group would allow the developer to set the whole group as "disabled", "readonly", vertically or horizontally laid out, types of display (like buttons, radio buttons, combobox, etc), and attributes set on the radio buttons would set their specific states, as we do with `<option>` tags. It is important to understand that the function of radio buttons is almost the same as a `<select>` tag, and this is why the comparison makes sense.
+The above anatomy is regular enough to be assembled in a way similar to (but more complex than) a `<select>`. Attributes set on the group would allow the developer to set the whole group as "disabled", "readonly", vertically or horizontally laid out, types of display (like buttons, radios, combobox, etc), and attributes set on the radios would set their specific states, as we do with `<option>` tags. It is important to understand that the function of radios is almost the same as a `<select>` tag, and this is why the comparison makes sense.
```

Comment:
I wouldn't bring `<select>` at this stage. Possibly consider a section at the end with something like, "Other solutions considered"